src/windowsiot/azext_windowsiot/generated/custom.py

```diff
@@ -0,0 +1,71 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+
+def windows_iot_services_list(client,
+                              resource_group_name=None):
+    if resource_group_name:
+        return client.list_by_resource_group(resource_group_name=resource_group_name)
+    return client.list()
+
+
+def windows_iot_services_show(client,
+                              resource_group_name,
+                              device_name):
+    return client.get(resource_group_name=resource_group_name,
+                      device_name=device_name)
+
+
+def windows_iot_services_create(client,
+                                resource_group_name,
+                                device_name,
+                                quantity,
+                                billing_domain_name,
+                                location=None,
+                                notes=None,
+                                admin_domain_name=None):
+    device_service = {}
+    device_service['location'] = location
+    device_service['notes'] = notes
+    device_service['quantity'] = quantity
+    device_service['billing_domain_name'] = billing_domain_name
+    device_service['admin_domain_name'] = admin_domain_name
+    return client.create_or_update(resource_group_name=resource_group_name,
```

Comment:
this extension only has track1 SDK?

Reply:
Good point!
I didn't add `--compatible-level=track2` when use codegen command. It should default to `track2` according to autorest.az user guide: https://github.com/Azure/autorest.az/blob/master/doc/how-to-generate-with-different-options.md#--compatible-level-option
@qiaozha Could you help take a look?

Reply:
For new RP, we could use track 2 SDK by default.

Reply:
Even if I retry with `--compatible-level=track2`, the generated sdk still seems to be track1......@qiaozha

Reply:
@Juliehzl This is track2 sdk actually. The function name is not `begin_create_or_update` because it's not long running operation from swagger definition.